### PR TITLE
Package Dependency Updates

### PR DESCRIPTION
**Pull request by cruzjone-flexion**
Successful Circle Deploy with Migration: [LINK](https://app.circleci.com/pipelines/github/flexion/ef-cms/46730/workflows/1dbce5c1-c5f1-4272-ad34-ab2256481474)

### CODE REVIEW

**Review by zachrog — APPROVED** [review]

**Review by rachelschneiderman — APPROVED** [review]
I think Cody mentioned the `aws` provider in our terraform files has been inadvertently skipped during dependency updates. Would you mind doing that and adding it to the doc for future folks?

File: iam/terraform/account-specific/main/main.tf [hunk 1]
```diff
@@ -17,6 +17,6 @@ terraform {
   }
 
   required_providers {
-    aws = "3.76.0"
+    aws = "5.16.1"
```
Comment (Absolutestunna) [inline iam/terraform/account-specific/main/main.tf:20]: 👏🏿

Comment (ujahio) [inline iam/terraform/account-specific/main/main.tf:20]: 👏🏿 👏🏿 
Thanks for this update. 
If you're doing this already, ignore, but please add instructions/reminders to the dependency updates docs to check/update `aws` versions.

File: iam/terraform/account-specific/main/circle-ci.tf [hunk 2]
```diff
@@ -326,3 +308,40 @@ resource "aws_iam_policy" "circle_ci_policy" {
...
 EOF
 }
+
+resource "aws_iam_user_policy_attachment" "circle_ci_route52_policy_attachment" {
+  user       = aws_iam_user.circle_ci.name
+  policy_arn = aws_iam_policy.circle_ci_route52_policy.arn
+}
+
+resource "aws_iam_policy" "circle_ci_route52_policy" {
+  name = "circle_ci_route52_policy"
```
Comment (mmarcotte) [inline iam/terraform/account-specific/main/circle-ci.tf]: hmm, perhaps a typo? Should this be `route53`?

**Review by mmarcotte — CHANGES_REQUESTED** [review]
I think just the `route52` adjustment is required.

File: iam/terraform/account-specific/main/circle-ci.tf (hunk 2 above)
Comment (cruzjone-flexion) [reply iam/terraform/account-specific/main/circle-ci.tf]: OK will fix it ASAP

**Review by ujahio — APPROVED** [review]

**Review by mmarcotte — APPROVED** [review]

**Pull request merged**